doc/developer-notes.md

```diff
@@ -603,10 +603,10 @@ Threads
   : Started from `main()` in `bitcoind.cpp`. Responsible for starting up and
   shutting down the application.
 
-- [ThreadImport (`b-loadblk`)](https://doxygen.bitcoincore.org/init_8cpp.html#ae9e290a0e829ec0198518de2eda579d1)
+- [ThreadImport (`b-loadblk`)](https://doxygen.bitcoincore.org/namespacenode.html#ab4305679079866f0f420f7dbf278381d)
```

Comment:
These URLs seem to be fragile. What about dropping them altogether and mentioning just the function name in the doc file.

In general, if it is not auto-generated from the source code then it will likely get outdated.

Reply:
if [this](https://stackoverflow.com/a/10631357/5800072) is still true, links are based on the function signature, and those functions have indeed change since the last edit on this doc.

Reply:
ok, so keeping it as is then